R/interact.R

```diff
@@ -162,7 +162,8 @@ prep.step_interact <- function(x, training, info = NULL, ...) {
     )
     if (!is_formula(tmp_terms)) {
       cli::cli_abort(
-        "{.arg terms} must be a formula. Not {.obj_type_friendly {term}}."
+        "{.arg terms} must be a formula. Not {.obj_type_friendly {term}}.",
+        .internal = TRUE
```

Comment:
I have in a couple of places added `.internal = TRUE`. I did this because I was not able to find a reprex, but it felt reasonable that it could happen.